packages/mdc-slider/index.js

```diff
@@ -162,10 +173,39 @@ class MDCSlider extends MDCComponent {
     this.max = parseFloat(this.thumb_.getAttribute(strings.ARIA_VALUEMAX)) || this.max;
     this.step = parseFloat(this.thumb_.getAttribute(strings.DATA_STEP)) || this.step;
     this.value = origValueNow || this.value;
+    if (this.tickMarkSet_) {
```

Comment:
Also check that the `mdc-slider--discrete` class exists, to enforce that tick marks are only allowed for discrete sliders.

Reply:
👍